pkg/localmetrics/localmetrics.go

```diff
@@ -270,3 +282,71 @@ func (c *MetricsCollector) AddAccountReuseCleanupFailure() {
 func (c *MetricsCollector) SetReconcileDuration(controller string, duration float64) {
 	c.reconcileDuration.WithLabelValues(controller).Observe(duration)
 }
+
+func (c *MetricsCollector) AddAPICall(controller string, req *http.Request, resp *http.Response) {
+	c.apiCallCount.With(prometheus.Labels{
+		"controller": controller,
+		"method":     req.Method,
+		"resource":   resourceFrom(req.URL),
+		"status":     resp.Status,
+	}).Inc()
```

Comment:
This can be simplified to `c.apiCallCount.WithLabelValues(controller, req.Method,resourceFrom(req.URL), resp.Status)`

Reply:
I don't see that as "simplified". I like the explicitness of listing the label names, rather than relying on the order they happen to be listed on L146.

Reply:
Umm, make sense!